packages/styling/src/styles/fonts.ts

```diff
@@ -37,7 +37,7 @@ export namespace LocalizedFontFamilies {
   export const Selawik = `'${LocalizedFontNames.Selawik}'`;
   export const Thai = `'Leelawadee UI Web', 'Kmer UI'`;
   export const Vietnamese = `'${LocalizedFontNames.Vietnamese}'`;
-  export const WestEuropean = `'${LocalizedFontNames.WestEuropean}'`;
+  export const WestEuropean = `'Segoe UI', '${LocalizedFontNames.WestEuropean}'`;
```

Comment:
Hmm, this probably breaks things, let me look into the local() option instead for the default font